Documentation/index.rst

```diff
@@ -53,17 +53,17 @@ Here is an example:
 
 .. code-block:: php
 
-  	$mail = new \Neos\SwiftMailer\Message();
+  $mail = new \Neos\SwiftMailer\Message();
```

Comment:
TIL, we have a separate RTD page for this package (https://swiftmailer-for-flow.readthedocs.io/en/latest/#installation) - is that really necessary? I would recommend moving the .rst as .md to the root and close that RTD page.